tyk-stack/values.yaml

```diff
@@ -189,6 +189,11 @@ global:
   # Choose the storageType for Tyk. [ "mongo", "postgres" ]
   storageType: &globalStorageType postgres
 
+  # hashKeys specifies that if your Tyk Gateway is using hashed keys, set this value to true so it matches.
+  # The Dashboard will now operate in a mode that is compatible with key hashing.
+  # It is used to set TYK_DB_HASHKEYS
```

Comment:
Can you also mention that this will take precedence over `dashboard.hashKeys` and will enable hashKeys for all the components.